src/main/java/ladder/model/LadderSize.java

```diff
@@ -0,0 +1,17 @@
+package ladder.model;
+
+public record LadderSize(int height, int width) {
+    public LadderSize {
+        validate(height, width);
+    }
+
+    private void validate(int height, int width) {
+        if (!isNaturalNumber(height) || !isNaturalNumber(width)) {
```

Comment:
특정 메서드가 계속해서 `!` 표시를 달아서 부정한다면 아예 메서드 안으로 넣고 `not` 표현을 추가해보는 건 어떨까요?
```suggestion
        if (notNaturalNumber(height) || notNaturalNumber(width)) {
```
이건 사람들마다 의견이 다 다른 부분이라 수정하지 않으셔도 괜찮아요
고민해보는 시간을 가져보죠!

Reply:
인텔리제이에서 `bool 메서드가 항상 반전됩니다` 라는 경고가 나왔지만 어떤 맥락인지 파악을 못 했었는데 이런 말이였군요. 개인적으로는 본문에서 예외를 직접 던지는 if문의 조건절은 앞에 `!` 연산해 주는 것을 선호하는데, 경고 표시가 거슬려서 `not~~()` 형태로 바꿔버렸습니다ㅎㅎ..